### PR TITLE
diffr: update to 0.1.4.

### PR DESCRIPTION
**Pull request by atweiden**

### CODE REVIEW

**Comment by atweiden** [comment]
Broken on 32-bit?

**Comment by Piraty** [comment]
please report this upstream, as it build fine previously

**Comment by atweiden** [comment]
> ~please report this upstream, as it build fine previously~
> please apply upstream's patch [mookid/diffr@eca5511](https://github.com/mookid/diffr/commit/eca551190affd4ce676bb184cd8cc52e20947bfc)

- [X] Done

File: srcpkgs/diffr/patches/eca551190affd4ce676bb184cd8cc52e20947bfc.diff [hunk 1]
```diff
@@ -0,0 +1,68 @@
+diff --git a/src/main.rs b/src/main.rs
```
[the remark this answers is not in the document]
Comment (atweiden) [reply srcpkgs/diffr/patches/eca551190affd4ce676bb184cd8cc52e20947bfc.diff]: ok

**Pull request merged**